_data/team-members.yml

```
@@ -210,3 +210,11 @@ members:
     photo: /resources/img/jedrzej.jpg
     contact_for: [scastie]
     active: true
+  - member: quentin
+    name: Quentin Bernet
+    title: Software Development - Intern
+    email: quentin.bernet@epfl.ch
+    github: Sporarum
+    photo: /resources/img/quentin.jpeg
+    contact_for: [Dotty, Documentation Improvements]
```

```suggestion
    contact_for: [dotty, documentationImprovements]
```

These values have to match the labels defined here: https://github.com/scala/scala.epfl.ch/tree/main/_projects